src/Coordination/CoordinationSettings.h

```diff
@@ -26,6 +26,7 @@ struct Settings;
     M(Milliseconds, heart_beat_interval_ms, 500, "Heartbeat interval between quorum nodes", 0) \
     M(Milliseconds, election_timeout_lower_bound_ms, 1000, "Lower bound of election timer (avoid too often leader elections)", 0) \
     M(Milliseconds, election_timeout_upper_bound_ms, 2000, "Upper bound of election timer (avoid too often leader elections)", 0) \
+    M(Milliseconds, leadership_expiry_ms, 0, "How long will a leader expire after not getting enough peer responses. Set it lower or equal to election_timeout_lower_bound_ms to avoid multiple leaders.", 0) \
```

Comment:
```suggestion
    M(Milliseconds, leadership_expiry_ms, 0, "Duration after which a leader will expire if it fails to receive responses from peers. Set it lower or equal to election_timeout_lower_bound_ms to avoid multiple leaders.", 0) \
```